web/cypress/integration/ssl/ssl-smoketest.spec.js

```diff
@@ -0,0 +1,125 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+/* eslint-disable no-undef */
+
+context('ssl smoke test', () => {
+  const domSelectors = {
+    notificationDesc: '.ant-notification-notice-description',
+    notificationMsg: '.ant-notification-notice-message'
+  };
+
+  beforeEach(() => {
+    // init login 
+    cy.login();
+  })
+
+  it('set match certificate and key by input', () => {
+    // go to ssl create page
+    cy.visit('/');
+    cy.contains('SSL').click();
+    cy.wait(500);
+    cy.contains('Create').click();
+
+    cy.get('#cert').type(`-----BEGIN CERTIFICATE-----
```

Comment:
too ugly here. need to extract public variables.

Reply:
good point, fixed.